src/Umbraco.Tests/Testing/UmbracoTestBase.cs

```diff
@@ -64,6 +65,7 @@ namespace Umbraco.Tests.Testing
     /// provides all the necessary environment, through DI. Yes, DI is bad in tests - unit tests.
     /// But it is OK in integration tests.
     /// </remarks>
+    [Apartment(ApartmentState.STA)]
```

Comment:
Is this necessary? This seems to indicate that we will prohibit async threads?

Reply:
I guess not, it was part of my fight with AsyncLocal - thanks.